icons/save-check.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Konixy"
```

You should include the people from `save` icon as well.
```suggestion
    "Konixy"
```